crates/viewer/re_viewer/src/app.rs

```diff
@@ -1413,14 +1411,24 @@ impl App {
                 .send_system(SystemCommand::set_selection(item.clone()));
         }
 
+        let mut time_commands = Vec::new();
+        if let Some(time_selection) = time_selection {
+            time_commands.push(TimeControlCommand::SetActiveTimeline(
+                *time_selection.timeline.name(),
+            ));
+            time_commands.push(TimeControlCommand::SetTimeSelection(time_selection.range));
```

Comment:
I think we should also turn on looping of the time selection, to further highlight "this is what the link wanted you to see"